operate/bridge/providers/relay_bridge_provider.py

```diff
@@ -0,0 +1,445 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2024 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+"""Relay Bridge provider."""
+
+
+import enum
+import json
+import math
+import time
+import typing as t
+from http import HTTPStatus
+from urllib.parse import urlencode
+
+import requests
+
+from operate.bridge.providers.bridge_provider import (
+    BridgeProvider,
+    BridgeRequest,
+    BridgeRequestStatus,
+    DEFAULT_MAX_QUOTE_RETRIES,
+    MESSAGE_EXECUTION_FAILED,
+    MESSAGE_QUOTE_ZERO,
+    QuoteData,
+)
+from operate.operate_types import Chain
+
+
+GAS_ESTIMATE_FALLBACK_ADDRESS = "0x000000000000000000000000000000000000dEaD"
+RELAY_DEFAULT_GAS = {
```

Comment:
did you find these constants in the docs? or explorer maybe? Please comment a link if it's documented anywhere.

Reply:
Commented on the code.